Demo/SwiftUI/PaymentsViewModel.swift

```diff
@@ -8,10 +8,10 @@ import SwiftUI
 
 internal final class PaymentsViewModel: ObservableObject, Identifiable, PresenterExampleProtocol {
 
-    private lazy var dropIn: DropInExample = {
-        let dropIn = DropInExample()
-        dropIn.presenter = self
-        return dropIn
+    private lazy var dropInAdvancedFlow: DropInAdvancedFlowExample = {
+        let dropInAdvancedFlow = DropInAdvancedFlowExample()
+        dropInAdvancedFlow.presenter = self
+        return dropInAdvancedFlow
```

Comment:
You need both `dropInAdvancedFlow` and `dropIn` 🤔

Reply:
I do not think so because we did not had session condition here in the old code as well. Also, for components e.g iDEAL, Card etc. we had no session condition.

Reply:
Ok, good point